packages/next/src/client/components/react-dev-overlay/app/client-entry.tsx

```diff
@@ -31,26 +30,12 @@ export function createRootLevelDevOverlayElement(reactEl: React.ReactElement) {
 
   socket.addEventListener('message', handler)
 
-  const FallbackLayout = hasMissingTags
-    ? ({ children }: { children: React.ReactNode }) => (
-        <html id="__next_error__">
-          <body>{children}</body>
-        </html>
-      )
-    : React.Fragment
-
   return (
-    <FallbackLayout>
-      <AppDevOverlay
```

Comment:
We shouldn't render another `AppOverlay` which is already rendered in the `HotReloader`,  we just need to have the boundary here to avoid here keep erroring